app/controllers/concerns/authorization.rb

```diff
@@ -0,0 +1,15 @@
+module Authorization
+  extend ActiveSupport::Concern
+
+  included do
+    include Pundit
+    rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+  end
+
+  private
+
+  def user_not_authorized
+    flash[:alert] = "You are not authorized to perform this action."
```

Comment:
We can use locales here, see documentation here https://guides.rubyonrails.org/i18n.html